### ISSUE TITLE
Failed installation due to outdated chokidar dependency

### ISSUE THREAD

**Issue by mistadikay** [Unitech/pm2#1646]
`pm2` installation fails because `node-gyp` fails to rebuild outdated `fsevents` package which is a dependency of outdated `chokidar`. Upgrading `chokidar` dependency to `~1.1.0` should solve this problem.

Installation log is below (`node@4.1.1`/`npm@3.3.5`):
```shell
npm ERR! Darwin 15.0.0
npm ERR! argv "/usr/local/Cellar/nvm/0.26.1/versions/node/v4.1.1/bin/node" "/usr/local/opt/nvm/versions/node/v4.1.1/bin/npm" "i" "pm2" "-g"
npm ERR! node v4.1.1
npm ERR! npm  v3.3.5
npm ERR! file sh
npm ERR! code ELIFECYCLE
npm ERR! errno ENOENT
npm ERR! syscall spawn

npm ERR! fsevents@0.3.8 install: `node-gyp rebuild`
npm ERR! spawn ENOENT
npm ERR!
npm ERR! Failed at the fsevents@0.3.8 install script 'node-gyp rebuild'.
```

**Comment by Unitech** [comment]
fixed in dev, will be published in 0.15.7 (tomorrow)

**Comment by mistadikay** [comment]
Great, thanks!
